src/applications/vaos/containers/DateTimeRequestPage.jsx

```diff
@@ -70,8 +76,21 @@ export class DateTimeRequestPage extends React.Component {
   };
 
   goForward = () => {
+    this.props.routeToNextAppointmentPage(this.props.router, pageKey);
+  };
+
+  validate = () => {
     if (this.props.data.calendarData?.selectedDates?.length) {
-      this.props.routeToNextAppointmentPage(this.props.router, pageKey);
+      this.setState({ validationError: null });
```

Comment:
Could we instead clear this when the user selects a date (the onChange prop)? That fits with the existing validation patterns on the site better.